code/go/0chain.net/blobbercore/reference/objectpath.go

```diff
@@ -89,3 +92,153 @@ func GetObjectPath(ctx context.Context, allocationID string, blockNum int64) (*O
 	//rootRef.CalculateHash(ctx, false)
 	return &retObj, nil
 }
+
+// TODO needs to be refactored
```

Comment:
How do we mark code that needs to be refactored through the codebase? I dont think we should create an issue every time for every function.
The new GetObjectPathGRPC function is just a copy of the old GetObjectPath function which is now compatible with grpc objects.
Refactoring the current code is beyond the scope of this migration PR.

Reply:
If you have a specific refactor in mind you should explain the TODO, like you just did, otherwise, it will just get ignored. `needs to be refactored` could mean anything.

Reply:
ok will add add more details to the comment